--- vte-0.64.0/src/vtedefines.hh	2021-04-19 22:55:33.665520166 +0000
+++ vte-0.64.0/src/vtedefines.hh~	2021-04-16 17:22:42.000000000 +0000
@@ -73,7 +73,7 @@
 #define VTE_CURSOR_FG                   262
 #define VTE_PALETTE_SIZE		263
 
-#define VTE_SCROLLBACK_INIT		512
+#define VTE_SCROLLBACK_INIT		4096
 #define VTE_DEFAULT_CURSOR		std::string{"text"}
 #define VTE_MOUSING_CURSOR		std::string{"default"}
 #define VTE_HYPERLINK_CURSOR		std::string{"pointer"}
